githublabels.py

```diff
@@ -10,12 +10,14 @@ def get_dpg_pog(): return []
   "rejected": "e11d21",
   "info": "0000ff",
   "doc": "257fdb",
+  "manually-approved": "defb04",
```

Comment:
@iarspider , I would suggest to use existing `info` so this label

Reply:
done